pkg/virt-handler/vm.go

```diff
@@ -1938,7 +1938,7 @@ func (d *VirtualMachineController) checkVolumesForMigration(vmi *v1.VirtualMachi
 				return blockMigrate, err
 			}
 			if !shared {
-				return true, fmt.Errorf("cannot migrate VMI with non-shared PVCs")
+				return true, fmt.Errorf("cannot migrate VMI: Live migration requires that all PVCs must be shared (using RWX access mode)")
```

Comment:
Can we mention the PVC which is causing the issue in the error?

Reply:
good idea

Reply:
done